paasta_tools/setup_kubernetes_cr.py

```diff
@@ -216,6 +219,22 @@ def setup_custom_resources(
     return succeded
 
 
+def get_dashboard_url(
+    kind: str, service: str, instance: str, cluster: str
+) -> Optional[str]:
+    if kind == "Flink":
```

Comment:
is the kind actually capitalized?

Reply:
Yes, I checked at the source. However you do have a point - this should be case-insensitive, I will fix.